elyra/cli/pipeline_app.py

```diff
@@ -311,5 +310,69 @@ def run(pipeline_path):
     print_banner("Elyra Pipeline Local Run Complete")
 
 
+@click.command()
+@click.option('--json',
+              'json_option',
+              is_flag=True,
+              required=False,
+              help='Returns pipeline file in a machine-readable JSON format')
+@click.argument('pipeline_path')
+def describe(json_option, pipeline_path):
+    """
+    Display pipeline description in a human readable format
```

Comment:
We should reword the help text to remove ambiguity (description != pipeline description) and remove the reference to the output format because multiple formats (text/JSON) are supported.

```suggestion
    Display pipeline summary
```